packages/lib-classifier/src/plugins/tasks/survey/models/SurveyTask.js

```diff
@@ -44,6 +44,21 @@ const Survey = types.model('Survey', {
   thumbnails: types.maybe(types.string),
   type: types.literal('survey')
 })
+  .preProcessSnapshot(snapshot => {
+    if (snapshot.hasOwnProperty('alwaysShowThumbnails')) {
+      const newSnapshot = { ...snapshot }
+      // 'true' and 'false' are both true.
+      if (snapshot.alwaysShowThumbnails === 'false') {
+
+      }
+      if (snapshot.alwaysShowThumbnails === 'true') {
+        newSnapshot.alwaysShowThumbnails = true
+      }
+      newSnapshot.alwaysShowThumbnails = !!newSnapshot.alwaysShowThumbnails
+      return newSnapshot
+    }
+    return snapshot
+  })
```

Comment:
Since `alwaysShowThumbnails` is deprecated, a better fix might be to remove it here and replace it with `thumbnails`.